docs/tutorials/configure/pet-photographer.md

```diff
@@ -133,7 +133,8 @@ Then, include it in the conditional statement in your filter function:
          detections = await self.vision_service.get_detections(img)
 ```
 
-- If `from_dm_from_extra` is `true` and the data management service is the caller of your filter function, the vision service is called to obtain an image and store it within the `img` variable.
+- The Python-configured camera checks if the data management service is the caller of the filter function by using `from_dm_from_extra` to determine whether to store data.
```

Comment:
@kaywux are we worried about other services calling the function? We should probably add why this is needed a bit more

Reply:
@agreenb regarding the data management service calling the Python color filter's [`get_image`](https://github.com/viam-labs/modular-filter-examples/blob/cd2f79c52b98c3deafbd1e9794869744803d4428/pycolorfilter/color_filter.py#L57C25-L57C25) function